pkg/keystore/awsKeystore.go

```diff
@@ -0,0 +1,99 @@
+package keystore
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/hashicorp/vault/api"
+	"io"
+	"log"
+)
+
+var (
+	unsealKeysFile = "unseal-keys.json.enc"
+	rootTokenFile  = "root-token.enc"
+)
+
+type AwsKeystore struct {
+	bucketName string
+	kmsKeyId   string
+	s3Service  *s3.S3
```

Comment:
Just wondering: Is it better to use S3 with KMS than to use the AWS Secrets Manager?

Reply:
good point. I'll try it out with the secrets manager